app/database/db_build.sql

```diff
@@ -10,7 +10,8 @@ CREATE TABLE nuggets(
   title VARCHAR(50) NOT NULL,
   description TEXT,
   img_url VARCHAR(100),
-  author VARCHAR(30) NOT NULL
+  author VARCHAR(30) NOT NULL,
```

Comment:
Why is it so short? What if my name is: "Pablo Diego José Francisco de Paula Juan Nepomuceno María de los Remedios Cipriano de la Santísima Trinidad Ruiz y Picasso"